src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -1327,206 +1327,85 @@ public struct ParentOrigin
         /// Top
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        public static float Top
-        {
-            get
-            {
-                float ret = Interop.NDalicParentOrigin.ParentOriginTopGet();
-                if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
-                return ret;
-            }
-        }
+        public const float Top = 0.0f;
```

Comment:
This changes `Property` to `Field`. ACR required;

How about this? This way maintains the API as a `Property` and returns a constant value.
```C#
public static float Top => 0.0f;
```